setup.py

```diff
@@ -14,10 +14,26 @@
 
 from setuptools import find_packages, setup
 
+beam_gcp_requirements = [
```

Comment:
Before the merge of this PR, would you mind cleaning up the history?

Reply:
Done! cherry-picked the relevant commits from main